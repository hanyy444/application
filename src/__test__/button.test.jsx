import React from "react";
import { render, screen } from '@testing-library/react';

import Button from "../components/button/button.component";

describe('Test button component', () => {
    expect.assertions(2);
    test('renders without crashing', () => {
        render(<Button text='Sign In'/>)
        expect(screen.getByTestId("button")).toHaveTextContent('Sign In')
        expect(screen.getByTestId("button")).toMatchSnapshot()
    })
})
