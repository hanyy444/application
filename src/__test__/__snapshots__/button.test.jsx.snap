// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test button component renders without crashing 1`] = `
<button
  class="sc-bczRLJ hilTVQ"
  data-testid="button"
>
   
  Sign In
   
</button>
`;
